docs/yaml/yaml.go

```diff
@@ -68,6 +68,9 @@ func GenYamlTreeCustom(cmd *cobra.Command, dir string, filePrepender func(string
 			return err
 		}
 	}
+	if !cmd.HasParent() {
```

Comment:
What does this do?

Reply:
I'll have to look into this one again, I think it was needed when generating docs for plugins, but it's been a while 😓